db/fixtures/server_roles.csv

```diff
@@ -2,6 +2,7 @@ name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
 database_operations,Database Operations,0,false,region
 database_owner,Database Owner,1,false,database
+embedded_ansible,Embedded Ansible,1,false,zone
```

Comment:
I think that keeping this zone will introduce more tech debt (in the form of preventing the role from starting in multiple zones) than the trouble of possibly having the role fail over into a different zone.